microsoft-edge/devtools-guide-chromium/customize/extensions.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Extend DevTools with extensions
+description: Learn to build a Microsoft Edge extension that extends DevTools.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 09/30/2022
+---
+# Extend DevTools with extensions
+
+If the tools available in Microsoft Edge DevTools are not enough for your use cases, you can add new tools by installing Microsoft Edge extensions or building your own extensions.
+
+Microsoft Edge extensions can insert new tabs in the main toolbar of DevTools.
+
+
+<!-- ====================================================================== -->
+## Install extensions in DevTools
+
+Install extensions in Microsoft Edge by navigating to [Edge Add-ons](https://microsoftedge.microsoft.com/addons/). 
+
+These extensions often extend the Microsoft Edge user interface with new buttons and panels, but can also extend DevTools by adding new tabs in the main toolbar.
```

Comment:
i was confused by the distinction between Microsoft Edge vs DevTools, need to highlight that contrast more distinctly, in my experience.
```suggestion
These extensions often extend the Microsoft Edge user interface with new buttons and panels, but can also extend DevTools, within Microsoft Edge, by adding new tabs in the main toolbar.
```
what about in the Drawer - can you add to Drawer or can user move it to Drawer?